lib/std/fs/watch.zig

```diff
@@ -678,7 +678,7 @@ fn testWriteWatchWriteDelete(allocator: Allocator) !void {
     };
 
     // overwrite line 2
-    const file = try std.fs.cwd().openFile(file_path, .{ .read = true, .write = true });
```

Comment:
I'm starting to question if this needs `.read = true` too. Not quite sure how to run the tests to make sure.
```sh-session
$ ~/repos/zig/build/zig test --zig-lib-dir lib lib/std/fs/watch.zig
Test [1/1] test "write a file, watch it, write it again, delete it"... SKIP
0 passed; 1 skipped; 0 failed.
```
This clearly doesn't seem to be the way to go.